cronlock

```diff
@@ -189,7 +189,14 @@ fi
 if [ "${CRONLOCK_TIMEOUT}" -gt 0 ]; then
   # ubuntu: aptitude install timeout
   timeout="$(which timeout)"
-  timeoutcmd="timeout -s 9 ${CRONLOCK_TIMEOUT}"
+  [[ $(readlink $timeout) == "/bin/busybox" ]] && /bin/busybox | head -n 1 | egrep -q 'v1\.2[0-9]\.[0-9]+'
+  # if we are on alpine version between 2.6 and 3.9, use different timeout command structure
```

Comment:
If it’s not true, we will crash now because we work with errexit. Can we move the conditions straight into the if? That should work.

Reply:
Hm and read link isn’t available on OS X, not sure about other platforms..

Reply:
Yea that's and issue...

Reply:
Maybe I'm not seeing the bigger picture, but it seems that `cronlock` already uses `readlink` - https://github.com/kvz/cronlock/blob/master/cronlock#L141. However, there is an alternative and it is called `realpath` - please let me know if this one is available on MacOS.

Reply:
Okay I didn't know/remember we already use it. I don't deserve any award for that because we do advertise BSD compatibility. But I also guess then at least we don't need to fix the issue in this PR! 😄 

I think we can use readlink for now, fix the errexit thingy, and merge. Then later on see if we can replace readlink with something more inclusive / less requiring of installing dependencies.